code/modules/shuttle/custom_shuttle.dm

```diff
@@ -161,9 +192,25 @@
 	//This should really be accelleration, but its a 2d spessman game so who cares
 	if(calculated_mass == 0)
 		return FALSE
-	calculated_acceleration = (calculated_dforce*1000) / (calculated_mass*100)
+	calculated_acceleration = calculated_dforce / calculated_mass * CUSTOM_SHUTTLE_ACCELERATION_SCALE
```

Comment:
```suggestion
	calculated_acceleration = (calculated_dforce / calculated_mass) * CUSTOM_SHUTTLE_ACCELERATION_SCALE
```
Just to reduce ambiguity, since I believe / and * have the same precedence.

Reply:
\* (Actually, I see later on you are now dividing by 10 rather than multiplying by 10 when calculating mass, ignore if this change is intended)